packages/api/.env.default

```diff
@@ -50,12 +50,13 @@ DEV_GMAIL_APP_PASSWORD=
 CONTACT_FORM_EMAIL=?
 CONTACT_FORM_PASSWORD=?
 
-# Probably not needed for dev?
-DO_SPACES_ACCESS_KEY_ID=?
-DO_SPACES_SECRET_ACCESS_KEY=?
+
```

Comment:
What this new line about huh? 2 lines not enough? haha

Reply:
Ha! OK I'll remove the offending line when I push an update to the documentation 😉